### PR TITLE
Remove OrleansTaskScheduler.Instance

### PR DESCRIPTION
**Pull request by ReubenBond**
Plus an extra commit to consolidate the "TaskScheduler" test category with the "Scheduler" test category.

Working outside in for #467 :)

### CODE REVIEW

File: test/NonSiloTests/SchedulerTests/OrleansTaskSchedulerBasicTests.cs [hunk 1]
```diff
@@ -149,7 +149,7 @@ public void Sched_Task_TplFifoTest()
             orleansTaskScheduler.Stop();
         }
 
-        [Fact, TestCategory("Functional"), TestCategory("TaskScheduler")]
+        [Fact, TestCategory("Functional"), TestCategory("Scheduler")]
```
Comment (jason-bragg) [inline test/NonSiloTests/SchedulerTests/OrleansTaskSchedulerBasicTests.cs]: If these have no external dependencies, please add BVT category.

**Comment by jason-bragg** [comment]
LGTM, but if these tests have no external dependencies, they should be run in the CI builds.

File: test/NonSiloTests/SchedulerTests/OrleansTaskSchedulerBasicTests.cs (hunk 1 above)
Comment (ReubenBond) [inline test/NonSiloTests/SchedulerTests/OrleansTaskSchedulerBasicTests.cs]: Fixed in latest.

**Comment by gabikliot** [comment]
LGTM.

**Pull request merged**